expression/builtin_json.go

```diff
@@ -1038,6 +1038,8 @@ func (b *builtinJSONQuoteSig) evalString(row chunk.Row) (res string, isNull bool
 	j, isNull, err = b.args[0].EvalJSON(b.ctx, row)
```

Comment:
Is it better to call EvalString here?
And we should check the type of the argument in `getFunction`.